config/projects.yml

```diff
@@ -1,14 +1,14 @@
 admin-bundle:
   branches:
     master:
-      php: ['7.2']
+      php: ['7.3', '7.2']
```

Comment:
@greg0ire is this order correct? I want to use 7.3 for lint and docs

Reply:
I really don't know, sorry.

Reply:
Please reverse, 7.2 first.